adhocracy4/exports/views.py

```diff
@@ -84,14 +88,27 @@ def get_field_data(self, item, name):
         except TypeError:
             pass
 
+        # Return the get_field_display value for choice fields
+        get_field_display_method = getattr(
+            item, 'get_{}_display'.format(name), None)
+        if get_field_display_method and callable(get_field_display_method):
+            return get_field_display_method()
+
+        fieldtype = self._get_field_type(item, name)
+
+        if fieldtype and fieldtype == 'RichTextField':
```

Comment:
i don't like the approach to make the `get_field_data` even more complicated. i think it would be better to put logic like the html cleaning to the concrete get_X_data method instead.
maybe we could also remove the call to the `get_X_display` and the logic to handle dicts, too (especially as i realized, that that the call `return item['name']` should be `return item[name]` instead. which we never realized as the the export is never called on lists of dicts)

Reply:
I was thinking about this, but I was afraid, that we would then miss this at some point, because we forget or don't know that a field is a RichTextFiled

Reply:
but if it is the html encodings then the problem occurs for comments, too

Reply:
No, because it is not a RichTextField

Reply:
then, as guesse above, i don't get the problem that is fixed with this pr
if i enter `greater > test` as a comment it gets stored as `greater &gt; test` to the database and exportet as `greater &gt; test` in the comments